softdev.bib

```diff
@@ -1622,3 +1622,13 @@ @article{chen16robust
     biburl    = {http://dblp.uni-trier.de/rec/bib/journals/corr/ChenR16},
     bibsource = {dblp computer science bibliography, http://dblp.org}
 }
+
+@article{antoch97effect,
+    author = {Antoch, Jaromir and Huskova, Marie and Prášková, Zuzana},
```

Comment:
The nearby entries use the author style `firstname lastname` instead of `lastname, firstname`. Do we want to make them consistent? I don't think it makes a difference in the rendered PDF.